crypto/aes/aes.h

```diff
@@ -131,9 +130,7 @@ AES_RETURN aes_encrypt_key192(const unsigned char *key, aes_encrypt_ctx cx[1]);
 AES_RETURN aes_encrypt_key256(const unsigned char *key, aes_encrypt_ctx cx[1]);
 #endif
 
-#if defined( AES_VAR )
```

Comment:
It seems to me that this, along with the missing check of the return value [here](https://github.com/trezor/trezor-firmware/blob/2446006db4d414dc00acffa18f88053e20bcd208/crypto/aes/aesgcm.c#L105), could be dangerous. If you compile it without `AES_256` and call `gcm_init_and_key(key, 32, ctx)`, the key will be ignored, but the function returns `RETURN_GOOD`.